### PR TITLE
Setup live demo + first update with Smart actions prefilled with a specific value

### PR DESCRIPTION
**Pull request by Tolsa**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -6,6 +6,7 @@
     post '/actions/mark-as-live' => 'companies#mark_as_live'
     post '/actions/upload-legal-docs' => 'companies#upload_legal_docs'
     post '/actions/charge-credit-card' => 'customers#charge_credit_card'
+    post '/actions/charge-credit-card/values' => 'customers#charge_credit_card_values'
```
Comment (Tolsa) [inline config/routes.rb]: This needs to be kept for Smart action with prefilled value

File: Gemfile [hunk 2]
```diff
@@ -5,7 +5,7 @@ git_source(:github) do |repo_name|
   "https://github.com/#{repo_name}.git"
 end
 
-
+gem 'forest_liana'
```
Comment (SeyZ) [inline Gemfile:8]: Already exists, please remove this.

File: config/database.yml [hunk 3]
```diff
@@ -1,18 +1,29 @@
...
-  adapter: postgresql
-  database: forest_live_demo_rails
-  encoding: unicode
-  pool: 5
-  host: localhost
-  port: 5446
-  username: forest
-  password: secret
+  <<: *default
+  database: forest-live-demo-rails_development
```
Comment (SeyZ) [inline config/database.yml:29]: Should not change

File: config/initializers/forest_liana.rb [hunk 4]
```diff
@@ -1,9 +1,2 @@
 ForestLiana.env_secret = Rails.application.secrets.forest_env_secret
-ForestLiana.auth_secret = Rails.application.secrets.forest_auth_secret
-
-ForestLiana.integrations = {
-  stripe: {
-    api_key: ENV['STRIPE_SECRET_KEY'],
-    mapping: 'Customer.stripe_id'
-  }
-}
```
Comment (SeyZ) [inline config/initializers/forest_liana.rb:9]: Why do you remove the Stripe integration?

File: config/secrets.yml [hunk 5]
```diff
@@ -18,7 +18,7 @@
 # Environmental secrets are only available for that specific environment.
 
 development:
-  forest_env_secret: e551d7119f000f8cef5bea7dbe7c549774d8e2140a7e9ece0ea0cbe6ca748cff
+  forest_env_secret: e209866f811fa84105cfa2db6d20eabd8fc598fdfdedb864b61816f6c90449ae
```
Comment (SeyZ) [inline config/secrets.yml:21]: Pas changé

File: db/schema.rb [hunk 6]
```diff
@@ -10,113 +10,124 @@
...
   add_foreign_key "orders", "customers", name: "orders_customer_id_fkey"
+  add_foreign_key "orders", "deliveries"
   add_foreign_key "orders", "deliveries", name: "orders_delivery_id_fkey"
+  add_foreign_key "orders", "products"
   add_foreign_key "orders", "products", name: "orders_product_id_fkey"
+  add_foreign_key "transactions", "companies", column: "beneficiary_company_id"
   add_foreign_key "transactions", "companies", column: "beneficiary_company_id", name: "transactions_beneficiary_company_id_fkey"
+  add_foreign_key "transactions", "companies", column: "emitter_company_id"
   add_foreign_key "transactions", "companies", column: "emitter_company_id", name: "transactions_emitter_company_id_fkey"
 end
```
Comment (SeyZ) [inline db/schema.rb:133]: should not change at all

**Review by SeyZ — CHANGES_REQUESTED** [review]

**Pull request closed without merging**